sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_manage_custom_models_async.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_manage_custom_models_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to manage the custom models on your account. To learn
+    how to create and train a custom model, look at sample_train_unlabeled_model.py and
+    sample_train_labeled_model.py.
+USAGE:
+    python sample_manage_custom_models_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
```

Comment:
We're moving to call these API Keys rather than subscription keys.  For context, see [this internal discussion](https://teams.microsoft.com/l/message/19:88f2d9dea2344075b5d4bc34d82a3d1c@thread.skype/1584989517146?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1584989517146&teamName=Azure%20SDK&channelName=Service%20-%20Form%20Recognizer&createdTime=1584989517146).